Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
```

Comment:
I wonder whether it makes sense to use a light weight container like alpine

Reply:
we could and maybe should make this a smaller image, making it alpine would need further testing as its not a straight swap out (maybe not for this PR)